### PR TITLE
BED 4488 - Updated documentation links in admin pages

### PR DESCRIPTION
**Pull request by StephenHinck**
## Description
Added page descriptions and documentation links to most admin pages for improved user experience in finding assistance with use of each page.

## Motivation and Context
Improving the ability for users to understand what each page does in the application, and ability to get help on how to use it, makes the use of the product simpler and require a lower cognitive burden on our users.

## How Has This Been Tested?
Local environment. Validated both with and without data present on all pages.

## Screenshots (if appropriate):

image
image
image
image

## Types of changes

-   [X] Chore (a change that does not modify the application functionality)
-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [X] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [ ] Documentation updates are needed, and have been made accordingly.
-   [ ] I have added and/or updated tests to cover my changes.
-   [X] All new and existing tests passed.
-   [ ] My changes include a database migration.

### CODE REVIEW

File: cmd/api/src/test/integration/harnesses/synclapspassword.svg
Comment (StephenHinck) [inline cmd/api/src/test/integration/harnesses/synclapspassword.svg:1]: License file miss

**Review by rrobin-code — APPROVED** [review]
Additional copy and links appear correct

File: cmd/ui/src/views/QA/QA.tsx [hunk 1]
```diff
@@ -76,6 +76,11 @@ const QualityAssurance: React.FC = () => {
                         }}
                     />
                 }>
+                <Typography variant='body1'>
+                    <p>
+                        Understand the data collected within BloodHound broken down by environment and principal type.
+                    </p>
+                </Typography>
```
[the remark this answers is not in the document]
Comment (StephenHinck) [reply cmd/ui/src/views/QA/QA.tsx:83]: I included the <p> tags as making it a paragraph enforced trailing whitespace ahead of the next section. It felt cramped without it.

[the remark this answers is not in the document]
Comment (superlinkx) [reply cmd/ui/src/views/QA/QA.tsx:83]: We could actually add "paragraph=true" to the Typography element and that would have the same effect without needing the extra tag

**Review by superlinkx — APPROVED** [review]
Made a quick comment about the Typography API, I wouldn't worry about it right now, let's get this in while pipelines are being nice to us.

File: cmd/ui/src/views/QA/QA.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (StephenHinck) [reply cmd/ui/src/views/QA/QA.tsx:83]: I'm happy to make those changes if we'd prefer for code hygiene!

**Pull request merged**